editor/debugger/editor_debugger_inspector.cpp

```diff
@@ -285,3 +286,13 @@ String EditorDebuggerInspector::get_stack_variable(const String &p_var) {
 	}
 	return String();
 }
+
+void EditorDebuggerInspector::_update_ep(EditorProperty *ep) {
+	if (ep->get_edited_property_value().get_type() == Variant::FLOAT && ep->is_read_only()) {
+		double value = ep->get_edited_property_value();
+		if (!Math::is_finite(value)) {
+			Object::cast_to<EditorPropertyFloat>(ep)->set_allow_non_finite(true);
+		}
+	}
+	EditorInspector::_update_ep(ep);
+}
```

Comment:
Is it possible to specify this parameter when initializing the control? In the case of the debugger, this should not change.

Reply:
probably, but I think we will still need to override a bit of `EditorInspector`

Reply:
ok ignore the last message. 
Is it ok to add to `PropertyUsageFlags`?
I found a much a better solution that doesn't require changing `EditorInspector` but it does require adding `PROPERTY_USAGE_DEBUG` to `PropertyUsageFlags`

Reply:
I don't think adding a new usage flag is justified for this purpose. Most likely there are more local solutions.

Reply:
More local solutions do exist but adding a usage flag is a lot cleaner.
I pushed a commit so you can see how much cleaner it is.

Reply:
sorry the last message in hindsight is kinda rude. 
I should have asked what do you mean by a more local solution?